File: 2221-find-triangular-sum-of-an-array/2221-find-triangular-sum-of-an-array.py
class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        if(len(nums)==1):
            return nums[0]
        temp=[]
        
        for i in range(len(nums)-1):
            temp.append((nums[i]+nums[i+1])%10)
        
        return self.triangularSum(temp)

File: 2221-find-triangular-sum-of-an-array/2221-find-triangular-sum-of-an-array.cpp
class Solution {
public:
    int triangularSum(vector<int>& nums) {
        if(nums.size()==1){
            return nums[0];
        }
        vector <int> temp;
        for (int i=0;i<nums.size()-1;i++){
            temp.push_back((nums[i]+nums[i+1])%10);
        }
        return this->triangularSum(temp);
    }
};